MainModule/Server/Core/Logs.lua

```diff
@@ -39,21 +38,27 @@ return function(Vargs)
 		Chats = {};
 		Joins = {};
 		Script = {};
+		Replications = {};
+		NetworkOwners = {};
```

Comment:
Why are replications and entwork owners added. They were remvoed before and I suggest you remove them.